bugwarrior/services/deck.py

```diff
@@ -0,0 +1,184 @@
+import datetime
+import logging
+import pathlib
+import base64
+
+import pydantic
+from typing import List
+import requests
+import typing_extensions
+from dateutil.tz import tzutc
+
+from bugwarrior import config
+from bugwarrior.services import IssueService, Issue, ServiceClient
+
+log = logging.getLogger(__name__)
+
+
+class NextcloudDeckConfig(config.ServiceConfig, prefix='deck'):
+    service: typing_extensions.Literal['deck']
+    base_uri: pydantic.AnyUrl
+    username: str
+
+    # can be a password or an app-password
+    password: str = ''
```

Comment:
Regardless of the type of password, isn't it a required field? If so a default value should not be specified.